CantoboardFramework/Keyboard/View/KeyView.swift

```diff
@@ -277,7 +282,7 @@ class KeyView: HighlightableButton {
         let keyboardIdiom = keyboardState.keyboardIdiom
         
         if let mainTextColor = titleColor(for: .normal)?.resolvedColor(with: traitCollection) {
-            setTitleColor(mainTextColor.withAlphaComponent(mainTextColor.alpha * (1 - swipeDownPercentage)), for: .highlighted)
+            setTitleColor(mainTextColor.withAlphaComponent(mainTextColor.alpha * (1 - swipeDownPercentage * 2.5)), for: .highlighted)
```

Comment:
The bottom label actually disappear earlier:
![Simulator Screen Shot - iPad Pro (12 9-inch) (5th generation) - 2021-10-13 at 13 50 54](https://user-images.githubusercontent.com/53967830/137074767-8de54ab6-05f6-4112-84e2-eccb3e1faa48.png)

This will make the label disappear at around 40%.